src/char/inter.c

```diff
@@ -815,18 +815,42 @@ static int inter_config_read(const char* cfgName)
 }
 
 // Save interlog into sql
-int inter_log(char* fmt, ...)
-{
+// order of passed args:
+// activity, origin, target, guild pointer, castle_id
+int inter_log(int ArgNum, ...)
+{ 
 	char str[255];
-	char esc_str[sizeof(str)*2+1];// escaped str
 	va_list ap;
+	char* activity;
+	char* origin;
+	char* target;
+	int gid = 0;
+	int castle_id = 0;
+	int i;
```

Comment:
You could line all these up so each one doesn't have it's own declaration line. IE:
```c
char *activity, *origin, *target;
int gid = 0, castle_id = 0, i;
```